Runtime.msbuild

```diff
@@ -65,25 +65,24 @@
 
   <Target Name="RestorePackages" DependsOnTargets="DownloadNuGet">
     <ItemGroup>
-      <_NuGetPackagesAndSolutions Include="src\System.Net.Http.Formatting.NetCore\packages.config;
-          test\System.Net.Http.Formatting.NetCore.Test\packages.config;
-          Runtime.sln" />
+      <_NuGetPackagesAndSolutions Include="Runtime.sln" />
 
       <!-- Avoid restoring RuntimePortable.sln directly. -->
       <_NuGetPackagesAndSolutions Include="src\System.Net.Http.Formatting.NetCore\packages.config;
           test\System.Net.Http.Formatting.NetCore.Test\packages.config"
         Condition=" '$(BuildPortable)' == 'true' " />
-      <_ProjectsToRestore Include="src\System.Net.Http.Formatting.NetStandard\System.Net.Http.Formatting.NetStandard.csproj;
```

Comment:
another nit: this project is restored transitively and it's more reliable not to kick that off at the same time as the test project